ci/jenkins/jobs/macros.yaml

```diff
@@ -7,6 +7,13 @@
           jenkins-jobs update -r ci/jenkins/jobs
           rm ci/jenkins/jobs/defaults.yaml
 
+- builder:
+    name: builder-jenkins-backup
+    builders:
+      - shell: |-
+          rdiff-backup --remove-older-than 6M backup
+          rdiff-backup {rdiff_username}@{rdiff_ip}::{rdiff_path} backup
```

Comment:
Is it better to reverse the line 14, 15. In this way, it won't remove old backup if current backup fails.

Reply:
We may need to create multiple vm for both lab and cloud jenkins, which means they will have different IPs. Do you think we should add one more job in cloud yaml?

Reply:
Update, thanks.